src/communication/azext_communication/azext_metadata.json

```diff
@@ -1,3 +1,3 @@
 {
-    "azext.minCliCoreVersion": "2.49.0"
+    "azext.minCliCoreVersion": "2.55.0"
```

Comment:
```suggestion
    "azext.minCliCoreVersion": "2.55.0",
    "azext.isPreview": true
```

Reply:
As the extension is in preview (1.8.0b2), so you should set `azext.isPreview` to `true`